nested-types.ipynb

```diff
@@ -0,0 +1,915 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 25,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "column_3 = pa.ListViewArray.from_arrays(offsets=[4, 7, 0, 0, 3],\n",
+    "                                        sizes=[3, 0, 4, 0, 2],\n",
+    "                                        values=[0, -127, 127, 50, 12, -7, 25],\n",
+    "                                        mask=pa.array([False, True, False, False, False]))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[<pyarrow.Buffer address=0x4bdc4021000 size=1 is_cpu=True is_mutable=True>,\n",
+       " <pyarrow.Buffer address=0x4bdc4050680 size=20 is_cpu=True is_mutable=True>,\n",
+       " <pyarrow.Buffer address=0x4bdc4050700 size=20 is_cpu=True is_mutable=True>,\n",
+       " None,\n",
+       " <pyarrow.Buffer address=0x4bdc4020fc0 size=56 is_cpu=True is_mutable=True>]"
+      ]
+     },
+     "execution_count": 26,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "column_3.buffers()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<pyarrow.lib.ListViewArray object at 0x108077ee0>\n",
+       "[\n",
+       "  [\n",
+       "    12,\n",
+       "    -7,\n",
+       "    25\n",
+       "  ],\n",
+       "  null,\n",
+       "  [\n",
+       "    0,\n",
+       "    -127,\n",
+       "    127,\n",
+       "    50\n",
+       "  ],\n",
+       "  [],\n",
+       "  [\n",
+       "    50,\n",
+       "    12\n",
+       "  ]\n",
+       "]"
+      ]
+     },
+     "execution_count": 28,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "column_3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 30,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "array([   0, -127,  127,   50,   12,   -7,   25])"
+      ]
+     },
+     "execution_count": 30,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "values_buffer = column_3.buffers()[4]\n",
+    "np.frombuffer(values_buffer, dtype=\"int64\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Struct\n",
+    "\n",
+    "A struct is a nested type parameterized by an ordered sequence of types.\n",
+    "\n",
+    "* one child array for each field\n",
+    "* child arrays are independent and need not be adjacent to each other in memory\n",
+    "\n",
+    "One can think of an individual struct field as a key-item pair where the key is the field and the child array its items/values. Filed (key) is saved in the schema and the items of specific field (key) are saved in the child array value buffer.\n",
```

Comment:
```suggestion
    "One can think of an individual struct field as a key-value pair where the key is the field name and the child array its values. The field (key) is saved in the schema and the values of a specific field (key) are saved in the child array.\n",
```

I think "key-value" is the more common term (it makes some things a bit more ambiguous, though, because we already use "values" in many other contexts ..).

And the values are just the child array, not necessarily only the child array's values buffer (depending on the data type, the child array might not even have a values buffer)